packages/core/src/generators/utils/generate-test-project.ts

```diff
@@ -25,8 +25,9 @@ export async function GenerateTestProject(
   }
 
   const suffix = schema.testProjectNameSuffix || 'test';
-  const testRoot = schema.projectRoot + '-' + suffix;
-  const testProjectName = schema.projectName + '-' + suffix;
+  const separator = schema.pathScheme === 'nx' ? '-' : '.';
+  const testRoot = schema.projectRoot + separator + suffix;
```

Comment:
This will be kinda ugly too, but we should default to `Test` rather than `test` for the suffix when using dotnet path scheme. 

Perhaps:
```typescript
  const suffix = schema.testProjectNameSuffix || (pathScheme === 'nx' ? 'test' : 'Test');
```

I don't really like having the `||` and a ternary here though, that's a lot of complexity represented by the single line. A decent amount of stuff changes based on that too, so it may be easier if we changed to something like:
```typescript
let suffix: string;
let separator: string;
if (schema.pathScheme === 'nx') {
  separator = '-'
  suffix = schema.testProjectNameSuffix || 'test'
} else {
  separator = '.'
  suffix = schema.testProjectNameSuffix || 'Test'
}
```

I'm not really a fan of that either though... It'd look better if abstracted into a method.
```typescript
// ....
const { suffix, separator } = getPathPartsFromSchema(schema)
// ...

function getPathPartsFromSchema(schema) {
  if (schema.pathScheme === 'nx') {
   return {
    separator: '-'
    suffix: schema.testProjectNameSuffix || 'test'
    }
  } else {
    return {
      separator: '.'
      suffix: schema.testProjectNameSuffix || 'Test'
    }
  }
}
```